Px.Utils/Validation/SyntaxValidation/SyntaxValidation.cs

```diff
@@ -0,0 +1,345 @@
+﻿using PxUtils.Exceptions;
+using PxUtils.PxFile;
+using PxUtils.PxFile.Meta;
+using System.Reflection.PortableExecutable;
+using System.Runtime.CompilerServices;
+using System.Text;
+
+namespace PxUtils.Validation.SyntaxValidation
+{
+    /// <summary>
+    /// Provides methods for validating the syntax of a PX file.
+    /// </summary>
+    public static class SyntaxValidation
+    {
+        /// <summary>
+        /// Collection of custom validation functions to be used during validation.
```

Comment:
The intention that the user of this library can use this class to inject their own validation functions in the validator should be expressed more clearly in the documentation.